internal/envtest/environment.go

```diff
@@ -137,7 +145,7 @@ func New(uncachedObjs ...client.Object) *Environment {
 	}
 
 	// Set minNodeStartupTimeout for Test, so it does not need to be at least 30s
-	clusterv1.SetMinNodeStartupTimeout(metav1.Duration{Duration: 1 * time.Millisecond})
```

Comment:
this is commented out now, because `SetMinNodeStartupTimeout` has been moved to internal [package](https://github.com/kubernetes-sigs/cluster-api/blob/main/internal/webhooks/machinehealthcheck.go#L48) and I am not sure if we should have something similar in operator code or we can just drop it